### PR TITLE
[Quantum RPG] Add information on how to play

### PR DESCRIPTION
**Pull request by dstrain115**
- Add more to the README about how to install and play the game.

### CODE REVIEW

File: unitary/examples/quantum_rpg/README.md [hunk 1]
```diff
@@ -1,11 +1,120 @@
...
+
+Each of these will have a status line that says how many qubits
+they have and in what state they are in.
+
+For instance, `1QP (0|1> 0|0> 1?)` means that the character has 1 qubit (1QP).
+None of the qubits have been measured in the 1 state (`0|1>`) or the 0 state
+(`0|1>`). 1 qubit is still unmeasured (`1?`).
+
+Once a character (or enemy) has measured half or more qubits in the zero state,
+they are **DOWN**.  If half or more are measured in the one state, they are
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/README.md:98]: DEL are
REPLACE one
WITH 1

File: unitary/examples/quantum_rpg/README.md [hunk 2]
```diff
@@ -1,11 +1,120 @@
...
+three BlueFoam enemies.
+
+Each of these will have a status line that says how many qubits
+they have and in what state they are in.
+
+For instance, `1QP (0|1> 0|0> 1?)` means that the character has 1 qubit (1QP).
+None of the qubits have been measured in the 1 state (`0|1>`) or the 0 state
+(`0|1>`). 1 qubit is still unmeasured (`1?`).
+
+Once a character (or enemy) has measured half or more qubits in the zero state,
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/README.md:97]: REPLACE zero
WITH 0

**Review by cognigami — CHANGES_REQUESTED** [review]

File: unitary/examples/quantum_rpg/README.md [hunk 3]
```diff
@@ -1,11 +1,120 @@
...
-This is a placeholder information page for the quantum RPG system and
-reference game: "Final State Preparation".  More information will be
-added here as we get closer to the official launch of the game in
-August 2023.
+This directory contains the rules for a Quantum RPG as well
+as a reference game titled "Final State Preparation".
+
+The overall idea is that characters and adversaries are quantum circuits.
+Each character and adversary will have actions that can modify (append)
+onto the circuit or measure the circuit.  The goal will be for the
```
[the remark this answers is not in the document]
Comment (dstrain115) [reply unitary/examples/quantum_rpg/README.md]: Done.

File: unitary/examples/quantum_rpg/README.md [hunk 4]
```diff
@@ -1,11 +1,120 @@
...
 
-This is a placeholder information page for the quantum RPG system and
-reference game: "Final State Preparation".  More information will be
-added here as we get closer to the official launch of the game in
-August 2023.
+This directory contains the rules for a Quantum RPG as well
+as a reference game titled "Final State Preparation".
+
+The overall idea is that characters and adversaries are quantum circuits.
+Each character and adversary will have actions that can modify (append)
```
[the remark this answers is not in the document]
Comment (dstrain115) [reply unitary/examples/quantum_rpg/README.md]: Done.

File: unitary/examples/quantum_rpg/README.md [hunk 5]
```diff
@@ -1,11 +1,120 @@
...
+The second part explains the rules and principles behind the Quantum RPG.
+
+## Final State Preparation
+
+
+### Running the game
+
+In order to run the game, clone the github repo, make sure that any requirements
+are installed with a command such as `pip install -r requirements.txt`
+
```
[the remark this answers is not in the document]
Comment (dstrain115) [reply unitary/examples/quantum_rpg/README.md]: Done.

File: unitary/examples/quantum_rpg/README.md [hunk 6]
```diff
@@ -1,11 +1,120 @@
...
+then run `python -m unitary.examples.quantum_rpg.main_loop`
+(you may need to make sure that the unitary library is in your PYTHONPATH).
```
[the remark this answers is not in the document]
Comment (dstrain115) [reply unitary/examples/quantum_rpg/README.md]: Done.

File: unitary/examples/quantum_rpg/README.md (hunk 2 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/README.md]: Replaced with |0>

File: unitary/examples/quantum_rpg/README.md (hunk 1 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/README.md]: Changed to |1>

**Review by cognigami — APPROVED** [review]

**Pull request merged**